cmd/delegation/commands/delegation.go

```diff
@@ -0,0 +1,200 @@
+package commands
+
+import (
+	"fmt"
+	"io/ioutil"
+	"math"
+	"os"
+	"sort"
+	"strconv"
+
+	gaia "github.com/cosmos/cosmos-sdk/cmd/gaia/app"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/x/auth"
+	"github.com/cosmos/cosmos-sdk/x/staking"
+	stakingtypes "github.com/cosmos/cosmos-sdk/x/staking/types"
+	"github.com/interchainio/delegation/pkg"
+	tmclient "github.com/tendermint/tendermint/rpc/client"
+
+	"github.com/spf13/cobra"
+)
+
+var (
+	cdc = gaia.MakeCodec()
+
+	// expects a locally running node
+	// TODO flag it up yo
+	node = tmclient.NewHTTP("https://rpc.cosmos.network:26657", "/websocket")
+
+	// expects the gos.json file to be here
+	gosJSON = "data/gos.json"
+
+	// output unsigned delegation tx
+	outputFile = "unsigned-delegations.json"
+)
+
+var RootCmd = &cobra.Command{
+	Use:   "delegation",
+	Short: "the delegation command",
+	Long:  "for something",
```

Comment:
ping @ebuchman. what should this say

Reply:
A tool for generating delegation transactions according to various strategies